apps/core/priv/scaffolds/terraform/providers/gcp.eex

```diff
@@ -16,15 +16,9 @@ terraform {
   }
 }
 
-locals {
-  gcp_location  = {{ .Values.Location | quote }}
-  gcp_location_parts = split("-", local.gcp_location)
-  gcp_region         = "${local.gcp_location_parts[0]}-${local.gcp_location_parts[1]}"
-}
-
 provider "google" {
   project = {{ .Values.Project | quote }}
-  region  = local.gcp_region
+  region  = {{ .Values.Region | quote }}
```

Comment:
Actually wait, since this value is not provided in old cli versions, there will be users who it will break for.  We should default it to `.Values.Location` if `.Values.Region` is not present

Reply:
Yeah that's a good idea.